qiita_pet/handlers/study_handlers/artifact.py

```diff
@@ -21,3 +25,72 @@ def get(self):
         artifact = to_int(self.get_argument('artifact_id'))
         self.write(artifact_graph_get_req(artifact, direction,
                                           self.current_user.id))
+
+
+class ArtifactAJAX(BaseHandler):
+    def get(self):
+        artifact_id = to_int(self.get_argument('artifact_id'))
+        self.write('TITLE FOR ARTIFACT %d' % artifact_id)
+
+    def post(self):
+        artifact_id = to_int(self.get_argument('artifact_id'))
+        self.write(artifact_delete_req(artifact_id, self.current_user.id))
+
+
+class ArtifactAdminAJAX(BaseHandler):
+    def get(self):
```

Comment:
That's a good point and that's the idea of templates. However, there might be a reason to leave them here, @squirrelo?